tasks/symdiff.js

```diff
@@ -72,11 +73,24 @@ module.exports = function(grunt) {
 
         // calculate the result
         var diff = symdiff(cssClasses, tplClasses, options.ignore);
-        if (diff.css.length) {
-            grunt.fail.warn('Unused CSS classes: ' + diff.css.join(' ') + '\n');
-        }
-        if (diff.templates.length) {
-            grunt.fail.warn('Unused template classes: ' + diff.templates.join(' ') + '\n');
+
+        ['CSS', 'Templates'].forEach(function(type) {
```

Comment:
Sorry about that, didn't even notice that.

Let me know if you want me to squash the commit.